tests/test_fingerprint.py

```diff
@@ -205,6 +207,78 @@ def func():
         hash3 = md5(datasets.utils.py_utils.dumps(create_ipython_func(co_filename, returned_obj))).hexdigest()
         self.assertEqual(hash1, hash3)
         self.assertNotEqual(hash1, hash2)
+        
+        co_filename, returned_obj = f"{tempfile}/ipykernel_12345/321456789.py", [0]
+        hash4 = md5(datasets.utils.py_utils.dumps(create_ipython_func(co_filename, returned_obj))).hexdigest()
+        co_filename, returned_obj = f"{tempfile}/ipykernel_12345/321456789.py", [1]
+        hash5 = md5(datasets.utils.py_utils.dumps(create_ipython_func(co_filename, returned_obj))).hexdigest()
+        co_filename, returned_obj = f"{tempfile}/ipykernel_12345/654123987.py", [0]
+        hash6 = md5(datasets.utils.py_utils.dumps(create_ipython_func(co_filename, returned_obj))).hexdigest()
+        self.assertEqual(hash4, hash6)
+        self.assertNotEqual(hash4, hash5)
```

Comment:
Cool ! Thanks for adding the tests :)

I think it's ok to have them here. Though I noticed you also added "test_dump_jupyter_function" below and copy pasted the already existing tests and therefore they're duplicated now